### DISCUSSION TITLE
第一次使用请问  我部署在vps上  想在本地机访问  如何改配置文件  试了很久没法 假如我的vpsip是x.x.x.x

### DISCUSSION THREAD

**Discussion by hibiki222** [stream-rec/stream-rec#132] [Q&A]
networks:
  stream-rec:

services:
  # 后端服务
  backend:
    # 使用最新的主分支镜像
    # 如果您想使用特定版本，请更改为streamrec/stream-rec:version
    # 您可以在https://hub.docker.com/r/streamrec/stream-rec/tags中找到所有可用的版本
    # latest是最新稳定内测版本，dev 是最新开发版。
    image: streamrec/stream-rec:latest
    # 容器名称，您可以将其更改为任何您喜欢的名称
    container_name: stream-rec
    restart: unless-stopped
    #  端口映射，您可以将端口更改为任何您喜欢的端口，默认为12555
    #  请注意，如果您更改了端口，请确保在前端服务中更改相关配置。
    ports:
      - "12555:12555"
    networks:
      - stream-rec
    volumes:
      # 将主机机器路径绑定到容器路径，存储路径映射
      # 下载路径，默认为当前运行目录下的downloads文件夹
      - "./downloads:/opt/records"
      # Rclone 配置文件路径绑定
      - "./rclone:/root/.config/rclone"
    environment:
      # 时区，默认为 Europe/Paris,您可以将其更改为任何您喜欢的时区，请确保前端服务中的时区与此处一致。
      - TZ=Asia/Shanghai
      # 代理设置，如果您在中国大陆，您可能需要设置代理。
      # 不需要代理的用户请将其注释掉。 只支持http代理。
      #- HTTP_PROXY=http://192.168.110.10:7890
      # 日志级别，默认为info。调试日志对于调试非常有用，但可能非常冗长。 遇到问题时，您可以将其更改为debug。
      - LOG_LEVEL=INFO
      # 容器数据库路径
      - DB_PATH=/opt/records
      # 容器第一次初始化下载路径
      - DOWNLOAD_PATH=/opt/records
      # 登录密钥，您可以将其更改为任何您喜欢的字符串, 用于登录
      # 请注意，该密钥只有第一次运行时有效，后续更改不会生效
      - LOGIN_SECRET=123

  # 前端服务
  frontend:
    #   使用最新的主分支镜像，请确保与后端服务tag一致
    # 如果您想使用特定版本，请更改为streamrec/stream-rec-front:version
    # 您可以在https://hub.docker.com/r/streamrec/stream-rec-front/tags中找到所有可用的版本
    # latest是最新稳定内测版本，dev 是最新开发版。
    image: streamrec/stream-rec-front:latest
    # 容器名称，您可以将其更改为任何您喜欢的名称
    container_name: stream-rec-frontend
    restart: unless-stopped
    #  端口映射，您可以将端口更改为任何您喜欢的端口，默认为15275
    ports:
      - "15275:15275"
    networks:
      - stream-rec
    depends_on:
      - backend
    environment:
      # 时区，默认为 Europe/Paris,您可以将其更改为任何您喜欢的时区。
      - TZ=Asia/Shanghai
      # 后端api url，由容器名称和端口组成。 如果您不知道自己在做什么，请不要更改它。
      - API_URL=http://backend:12555/api
      # 加密密钥，用于next-auth，您可以将其更改为任何您喜欢的字符串
      - NEXTAUTH_SECRET=21312312
      # 客户端访问url。 用于next-auth重定向登录页面。
      # 这应该使用部署服务器的前端地址。
      # 确保端口与前端端口相同。
      - NEXTAUTH_URL=http://0.0.0.0:15275/
      # WS_API_URL，客户端websocket url。 用于实时状态更新。
      # 这应该使用部署服务器的ip地址。
      # 例如，以下是使用localhost的示例，您可以将其更改为部署服务器的ip地址。
      # 例如，如果后端部署在12.12.12.12的服务器上，端口为12555，则应将其更改为 ws://12.12.12.12:12555/live/update
      - WS_API_URL=ws://http://0.0.0.0:12555/live/update

**Comment by hua0512** [comment]
WS_API_URL更改为ws://http://x.x.x.x:12555/live/update
NEXTAUTH_URL=http://x.x.x.x:15275/
